gulpfile.js

```diff
@@ -133,22 +130,42 @@ gulp.task(
     "copy-uswds-js",
     "build-sass"
   )
-);
+);*/
 
-gulp.task('update', gulp.series(
+/*gulp.task('update', gulp.series(
   'copy-uswds-core',
   'copy-uswds-lib',
   'copy-uswds-settings',
   'copy-uswds-fonts',
   'copy-uswds-images',
   'copy-uswds-js',
   'build-sass',
-));
+));*/
 
-gulp.task("watch-sass", function() {
-  gulp.watch(`${PROJECT_SASS_SRC}/**/*.scss`, gulp.series("build-sass"));
-});
 
-gulp.task("watch", gulp.series("build-sass", "watch-sass"));
+exports.init =gulp.series(
```

Comment:
```suggestion
exports.init = gulp.series(
```

Needs a space after the `=`.